docs/en/event-daemon/event-daemon-installation.md

```diff
@@ -15,9 +15,8 @@ The following guide will help you setup {% include product %}Events for your stu
 
 The daemon can run on any machine that has Python installed and has network access to your {% include product %} server. It does **not** need to run on the {% include product %} server itself. In fact, if you are using the hosted version of {% include product %}, this isn't an option. However, you may run it on your {% include product %} server if you like. Otherwise, any server will do.
 
-* Python v2.6, v2.7 or 3.7
 * [{% include product %} Python API](https://github.com/shotgunsoftware/python-api)
-  * Use v3.0.37 or higher for Python v2.6 or v2.7 and use v3.1.0 or more for Python 3.7.
+  * Use use v3.1.0 or more for Python 3.7 (_note: Python 2 is no longer supported_).
```

Comment:
SGD 1.8.0 is shipped with Python 3.9 too. But I don't know which Python is used in this context.

Reply:
Looks like Python 3.7+: https://github.com/shotgunsoftware/python-api/blob/master/shotgun_api3/lib/httplib2/python3/__init__.py